docs/recipes/hyperlinks.md

```diff
@@ -0,0 +1,78 @@
+# Use hyperlinks directly in the terminal
+
+This recipe is an example on how to leverage hyperlinks directly in the terminal.
+Hyperlinks are generated by terminal applications by using the [OSC-8 escape sequences](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda).
+
+## Requirements
+
+By default, `ls` doesn't generate hyperlinks, here is an example of aliases to enable them in some common tools:
+
+```sh
+alias ls='ls --hyperlink --color=auto'
+alias delta="delta --hyperlinks --hyperlinks-file-link-format='file://{path}#{line}'"
+alias rg='rg --hyperlink-format=kitty'
+```
+
+> On macOS, you'll need to install `ls` from coreutils. For example using [homebrew](https://formulae.brew.sh/formula/coreutils).
+
+## Configuration
+
+The following snippet enables directory navigation and will open any text file using `nvim`.
+
+```lua
+local wezterm = require("wezterm")
+local act = wezterm.action
+local config = wezterm.config_builder()
+
+wezterm.on("open-uri", function(window, pane, uri)
+  local editor = "nvim"
+
+  if uri:find("^file:") == 1 then
+    -- uri format is: file://[HOSTNAME]/PATH[#LINENR]
+    local url = wezterm.url.parse(uri)
+
+    -- figure out what kind of file we're dealing with
+    local success, stdout, _ = wezterm.run_child_process({ "file", url.file_path })
+    if success then
+      if stdout:find("directory") then
+        pane:send_text(
+          wezterm.shell_join_args({ "cd", url.file_path }) .. "\r" .. wezterm.shell_join_args({ "ls", "-a", "-p", "--group-directories-first" }) .. "\r"
+        )
+        return false
+      end
+
+      if stdout:find("text") then
+        if url.fragment then
+          pane:send_text(wezterm.shell_join_args({ editor, "+" .. url.fragment, url.file_path }) .. "\r")
+        else
+          pane:send_text(wezterm.shell_join_args({ editor, url.file_path }) .. "\r")
+        end
+        return false
+      end
+    end
+  end
+
+  -- without a return value, we allow default actions
+end)
+
+config.mouse_bindings = {
+    -- Trigger open selection (default is without a modifier)
+    {
+      event = { Up = { streak = 1, button = "Left" } },
+      mods = "SUPER",
+      action = act.OpenLinkAtMouseCursor,
+    },
+    -- Disable the 'Down' event of CTRL-Click to avoid weird program behaviors
```

Comment:
Note: The comment mentions CTRL, but the keybind uses SUPER..
What do you think about using CTRL instead for the keybind?